src/App.jsx

```diff
@@ -14,45 +17,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_BY_INITIAL = '';
+const SORT_BY_ALPHABET = 'name';
+const SORT_BY_LENGTH = 'length';
+
+function getPreparedGoods(goods, sortedBy, isReversed) {
+  const preparedGoods = goods.map(good => (
+    {
+      id: goods.indexOf(good),
```

Comment:
```suggestion
  const preparedGoods = goods.map((good, index) => (
    {
      id: index,
```
You can get the index from map callback. But it's not a good idea to set a new id for each good inside function. Would be great to prepare them outside of the function. Also, it's better to use more unique value than indexes. Consider using some tools like uuid for generating unique ids. Btw, in this task it's not critical, you can just use good's name as a key

Reply:
Yep, i like use `crypto.randomUUID()`, but how i see not all know about that, so i don't use that, BUT if i can - i do that with joy) (i will use uuid)